Flepper.QueryBuilder/Operators/Alias/AliasOperator.cs

```diff
@@ -14,8 +15,17 @@ public AliasOperator(StringBuilder command, IDictionary<string, object> paramete
 
         public IAliasOperator As(string alias)
         {
-            Columns = Columns.Select(c => c.TableAlias != null ? c : new SqlColumn($"[{alias}].{c}")).ToArray();
+            var withoutFunctions = Columns
+                .Where(c => !c.GetType().IsSubclassOf(typeof(FunctionOperator)))
+                .Select(c => c.TableAlias != null ? c : new SqlColumn($"[{alias}].{c}")).ToArray();
```

Comment:
Since you changed the SqlColumn constructor
Change this `new SqlColumn($"[{alias}].{c}")` for `new SqlColumn(c.ToString(), alias)`